### DISCUSSION TITLE
Verions 2.1.1 online

### DISCUSSION THREAD

**Discussion by beabel** [beabel/radiowecker#17] [Ankündigungen / Announcements]
Die neue Version hat die angefragte Funktion des Senderwechsel von der Website integriert.
Somit ist jetzt eine Nutzung auch komplett ohne Display möglich.
Es reicht ein ESP DEV, 1 oder 2 x MAX und Lautsprecher.

![Main Screen](https://github.com/beabel/radiowecker/assets/12296286/9dc333f1-36f0-4e80-9ce9-f30a52fc68b9)

minimal Version :
![Minimal Version](https://github.com/beabel/radiowecker/assets/12296286/57eb1361-4599-4d09-9593-497d5bce2aa1)

**Comment by brezel64** [comment]
danke für die neue Version, wirklich gut.
bestehend bleibt leider das die Uhrzeit im Display etwas nachhängt, jetzt aber über das Webinterface korrekt angezeigt wird.
In einer vorigen Version war der Beeper beim Wecken noch aktiv, jetzt nicht mehr.
Ist es möglich den Beeper für akustisches Feedback bei Tastendruck zu nutzen?

**Reply by beabel** [reply]
den Zeitversatz habe ich angepasst wie kingherold es beschrieben hat.
Sollte in der nächsten Version mit drin sein, momentan ist es nur im develop.
Den Beeper als Tastendruck ist sicherlich möglich, mich jedoch würde es nerven.
Mal schauen, ob ich es mit in das Webinterface integriere und auf Wunsch dann Tastentöne enthalten sind.

**Comment by KuhnDaniel** [comment]
Danke auch von meiner Seite. Wollte schon fragen nach einer Möglichkeit für Senderwechsel via Web :-) Da warst du schneller. Falls ein Gehäuse gewünscht ist, würde ich dir das Drucken und zusenden - als Geschenk - versteht sich. Für die tolle Arbeit. LG Daniel
